pinery-miso-dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM maven:3.6.0-jdk-8 as builder
+COPY . /miso-lims
+RUN cd miso-lims && mvn clean package
```

Comment:
Would be nice to have this run maven with `-q` to move the internet-downloading offscreen

Reply:
added this as well as `-B` (batch-mode)